src/Model.cpp

```diff
@@ -93,8 +93,10 @@ std::string Model::get_name() const
 
 void Model::draw() const
 {
-    for (auto const& mesh : m_meshes)
-        mesh->draw();
+    if (enabled) {
+        for (auto const& mesh : m_meshes)
+            mesh->draw();
+    }
```

Comment:
I'm wondering if this is the best thing to do. Because this doesn't remove the Drawable from the Renderer loop, which means that although a Drawable component is disabled and we don't render it, part of the render pipeline for this object still runs (we are still looping over it in Renderer, we still set all shaders and constant buffers for it).

There are new methods in `main` for exactly something like this, I'll look into that